lib/booking/ui/pages/room_pages/add_edit_room_page.dart

```diff
@@ -130,20 +218,67 @@ class AddEditRoomPage extends HookConsumerWidget {
                             ),
                           ],
                         ),
-                        child: Text(
-                            isEdit
-                                ? BookingTextConstants.edit
-                                : BookingTextConstants.add,
-                            style: const TextStyle(
-                                color: Colors.white,
-                                fontSize: 25,
-                                fontWeight: FontWeight.bold))),
-                  ),
+                        child: const CircularProgressIndicator(
+                          color: Colors.white,
+                        ),
+                      ),
+                      onTap: () async {
+                        await tokenExpireWrapper(ref, () async {
+                          await showDialog(
+                              context: context,
+                              builder: (context) => CustomDialogBox(
+                                    descriptions: BookingTextConstants
+                                        .deleteRoomConfirmation,
+                                    onYes: () async {
+                                      final value = await roomListNotifier
+                                          .deleteRoom(room);
+                                      if (value) {
+                                        QR.to(BookingRouter.root +
+                                            BookingRouter.admin);
```

Comment:
Can't you use QR.back() here ?

Reply:
Yes, I changed it to QR.back()
I also did the same for the add/edit button on the same page.